src/Symfony/Component/Form/Extension/Core/Type/FormType.php

```diff
@@ -226,7 +226,6 @@ public function configureOptions(OptionsResolver $resolver)
         $resolver->setAllowedTypes('label_attr', 'array');
         $resolver->setAllowedTypes('action', 'string');
         $resolver->setAllowedTypes('upload_max_size_message', ['callable']);
-        $resolver->setAllowedTypes('help', ['string', 'null']);
```

Comment:
This doesn't look right to me. We do not want to allow everything here.

Reply:
It would also work with 
````php
$resolver->setAllowedTypes('help', ['string', 'null', TranslatableMessage::class]);
````

But would we want to limit 'label' with the same allowed types?

Reply:
`$resolver->setAllowedTypes('help', ['string', 'null', TranslatableMessage::class]);` looks perfect to me. I wouldn't touch the `label` option here though. That could be discussed in a different PR.